deep_translator/base.py

```diff
@@ -2,10 +2,14 @@
 
 __copyright__ = "Copyright (C) 2020 Nidhal Baccouri"
 
+import asyncio
 from abc import ABC, abstractmethod
+from functools import lru_cache
 from pathlib import Path
 from typing import List, Optional, Union
 
+import aiohttp
```

Comment:
You will need a try catch here when you make this depedency optional or maybe just add the import inside the function where this will be used. For an example, check out the docx or pypdf dependencies.